plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/DispositionTab/DispositionTab.tsx

```diff
@@ -50,23 +72,45 @@ const DispositionTab = (props: OwnProps) => {
     if (isNotesEnabled() && notes) {
       payload.notes = notes;
     }
-
+    payload.custom_attributes = { ...customAttributes };
     dispatch(updateDisposition(payload));
   };
 
   const updateStoreDebounced = debounce(updateStore, 250, { maxWait: 1000 });
 
   useEffect(() => {
+    const getQueueFromInstantQuery = async (queueSid: string) => {
```

Comment:
Are we doing this look up just to get the queue name? If so can we remove it, the queue name is already on the task task.queueName, task.queueSid

Reply:
Done!